plugins/media/wowza/services/LiveConversionProfileService.php

```diff
@@ -152,15 +152,16 @@ public function serveAction($streamName, $hostname = null, $extraParams = null)
 					$defaultFrameRate = $liveParamsItem->getFrameRate();
 				}
 			}
-			$this->appendLiveParams($entry, $mediaServer, $encodes, $liveParamsItem, $streamParametersArray);
-			$tags = array("all");
-			foreach($tags as $tag)
-			{
-				if(!isset($groups[$tag]))
-					$groups[$tag] = array();
-
-				$systemName = $liveParamsItem->getSystemName() ? $liveParamsItem->getSystemName() : $liveParamsItem->getId();
-				$groups[$tag][] = $systemName;
+			$shouldAddFlavor = $this->appendLiveParams($entry, $mediaServer, $encodes, $liveParamsItem, $streamParametersArray);
+			if ($shouldAddFlavor) {
+				$tags = array("all");
```

Comment:
The 'all' value should be either a const value for this class , or a config value if there is an option for future change.